js/apps/admin-ui/src/dashboard/Dashboard.tsx

```diff
@@ -67,48 +69,46 @@ const EmptyDashboard = () => {
   );
 };
 
+const FeatureItem = ({ feature }: { feature: FeatureRepresentation }) => {
+  const { t } = useTranslation("dashboard");
+  return (
+    <ListItem key={feature.name} className="pf-u-mb-sm">
```

Comment:
There should be no need to specify the `key` here, since that is passed when this component is used.
```suggestion
    <ListItem className="pf-u-mb-sm">
```